egs2/aishell3/tts1/conf/tuning/finetune_tacotron2.yaml

```diff
@@ -0,0 +1,78 @@
+# This configuration is for ESPnet2 to finetune Tacotron 2. Compared to the
```

Comment:
This configuration is for fine-tuning of single speaker case.
If the recipe is multi-speaker, please replace the config with the following one.
3d83127/egs2/vctk/tts1/conf/tuning
maybe only using the following configs is OK: (default is tacotron2 or transformer)

- train_gst+xvector_conformer_fastspeech2.yaml
- train_gst+xvector_tacotron2.yaml
- train_gst+xvector_transformer.yaml